deploy/helm/sumologic/conf/buffer.output.conf

```diff
@@ -1,5 +1,4 @@
-compress gzip
 flush_interval "#{ENV['FLUSH_INTERVAL']}"
 flush_thread_count "#{ENV['NUM_THREADS']}"
 chunk_limit_size "#{ENV['CHUNK_LIMIT_SIZE']}"
```

Comment:
should we increase chunk_limit_size to 1mb knowing that the chunks will no longer be compressed?

Reply:
we do still do compression for memory buffer and file buffer without persistence, it might be really tricky to have two defaults depending on the buffer type..

Reply:
OK I'm fine with keeping this for now. Ideally we figure out how to continue to use gzip since these metrics will compress heavily (repeated dimensions + metadata)

Reply:
Could this be the issue we're seeing? https://github.com/fluent/fluentd/issues/2619

Have we tried upgrading our fluentd image from v1.6.3 to v1.8.1 and see if it resolves the issue?

Reply:
will try that

Reply:
yep, v1.8.1 fixed the issue! will revert the compression changes